app/models/resource.rb

```diff
@@ -1,6 +1,6 @@
 class Resource < ApplicationRecord
   include Elasticsearch::Model
-  index_name "articles-#{Rails.env}"
+  index_name "#{self.class.name.downcase}-#{Rails.env}"
```

Comment:
I'm totally into using slashes in the index names as long at they are allowed characters. It seems like [maybe they are not?](https://github.com/elastic/elasticsearch/issues/6736#issue-37167454).

Either way, I've pluralized in this commit: e0c9663.